Add Spotlights Renewal News Article (#3438)
ppy/osu-wiki: news/2020-04-10-osu-spotlights-new-team-and-league-system.md

---
layout: post

> title: "osu! Spotlights: New Team and League System"

Reviewer: could be mistakenly read as "new team system" which sounds like teams of players instead of the spotlights team

> date: 2020-04-13 08:00:00 +0000

Reviewer: date doesnt match filename

---

> With our latest [Spotlight season] released very recently, we are underway reworking the osu! Spotlights from grounds up, starting with a brand new team and adding a whole new league system.

Reviewer 2: Two different options, either one would work.  `the osu! Spotlights`  sounds weird because `osu! Spotlights` is a name (proper noun) not a normal noun. Either `the` should be removed, or `project` (or a similar noun) should be added.

![](image)

> Since March 2017 osu! Spotlights work in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea and give the curation and participation in the spotlights a whole new twist.

Reviewer 2: ```suggestion
Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving curation and participation in the Spotlights a whole new twist.
```
word tense + better flow for the second sentence (reduced usage of `and`)

> Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving the curation and participation in osu! Spotlights a whole new twist.

Reviewer: > by giving the curation and participation in osu! Spotlights a whole new twist.

this sounds like you're trying to introduce a surprise, but you've already mentioned the changes in the previous paragraph. i think it would be better to get to the point faster since your reader is already expecting to hear about new team and leagues

Author: Well, they just need to read onwards

> Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving the curation and participation in osu! Spotlights a whole new twist.

Reviewer: repeating "a whole new" doesn't have much effect other than making this tedious to read, I'd save that for the intro only

> While we want to keep recommending beatmaps to you, we also want to add a whole new competitive and seasonal league closely attached to the recommended beatmaps. With this you and your friends will have even more reason to delve deep into the best creations osu! has to offer.

Reviewer 2: ```suggestion
While we want to keep recommending beatmaps to you, we also want to add a whole new competitive and seasonal league closely attached to the recommended beatmaps. With this, you and your friends will have even more reason to delve deep into the best creations osu! has to offer.
```

> While we want to keep recommending beatmaps to you, we also want to add a whole new competitive and seasonal league closely attached to the recommended beatmaps. With this, you and your friends will have even more reason to delve deep into the best creations osu! has to offer.

Reviewer: single line breaks don't do anything, if this was supposed to be a new paragraph u need another break

> While we want to keep recommending beatmaps to you, we also want to add a whole new competitive and seasonal league closely attached to the recommended beatmaps. With this, you and your friends will have even more reason to delve deep into the best creations osu! has to offer.

Reviewer: same "while \_\_, we want \_\_" thing as the last sentence, again just feels tedious to read like this

> We are still working on the details on how this league will work in detail, but here are some aspects we _plan_ to do:

Reviewer 2: ```suggestion
We are still working on the details about how this league will work, but here are some aspects we _plan_ to do:
```
`details` are already `in detail` :^)

> We are still working on the details about how this league will work, but here are some aspects we *plan* to do:

Reviewer: ```suggestion
We are still working on the details of how this league will work, but here are some aspects we *plan* to do:
```

> - **We want to host weekly timeshift lobbies**, using osu!lazer's recently implemented feature, having people compete actively on a portion of the beatmaps we have selected as Spotlight Beatmaps.

Reviewer 2: ```suggestion
- **We want to host weekly timeshift lobbies**, using osu!lazer's recently implemented feature, by having people compete actively on a portion of the beatmaps we have selected for the Spotlight.
```

> - **We want to host weekly timeshift lobbies**, using osu!lazer's recently implemented feature, by having people compete actively on a portion of the beatmaps we have selected for the Spotlights.

Reviewer: ```suggestion
- **We want to host weekly timeshift lobbies** using osu!lazer to have people compete actively on a portion of the beatmaps we have selected for the Spotlights.
```
also not sure if introducing timeshift without explanation is a good idea. a link would be fine if an inline explanation doesn't belong (i guess it's time to explain this on the wiki anyway)

> - With hosting weekly multiplayer lobbies, **we will release our recommendations little by little**. To keep things fresh, maps being played in the lobby will change every week. Because of that we will only publish portions of the whole spotlights every week.

Reviewer 2: ```suggestion
- With hosting weekly multiplayer lobbies, **we will release our recommendations little by little**. To keep things fresh, maps being played in the lobby will change every week. Because of this, we will only publish portions of the whole Spotlight every week.
```

> - With hosting weekly multiplayer lobbies, **we will release our recommendations little by little**. To keep things fresh, maps being played in the lobby will change every week. Because of this, we will only publish portions of the whole Beatmap Spotlights every week.

Reviewer: first sentence seems redundant
```suggestion
- To keep things fresh, **maps being played in the lobby will change every week**. Those maps will come from a portion of the whole Beatmap Spotlights.
```

> - With hosting weekly multiplayer lobbies, **we will release our recommendations little by little**. To keep things fresh, maps being played in the lobby will change every week. Because of this, we will only publish portions of the whole Beatmap Spotlights every week.

Reviewer: is spotlights still seasonal then? will there be more spotlighted maps overall if new ones come every week? should have answers (even if they're tentative) to these questions in the post

> - When a timeshift lobby ends, we want to **award the winners of the timeshift lobbies weekly with a small prize**.

Reviewer: ```suggestion
- When a timeshift lobby ends, we want to **award the winners with a small prize**.
```

> - On top of that we also want to add a **Seasonal Ranking** where we will sum up the results of each timeshift lobby and pit every participant against another with a custom ranking system. We plan to assign a league bracket to every participant, based on their placement in this ranking system (Bronze, Silver, Gold, etc.). Once a season ends, maybe you can even keep your placement in shape of a medal or badge on your profile...?

Reviewer: ```suggestion
- On top of that, we also want to **add a Seasonal Ranking** where we will sum up the results of each timeshift lobby and pit every participant against another with a custom ranking system. We plan to assign a league bracket to every participant, based on their placement in this ranking system (Bronze, Silver, Gold, etc.). Once a season ends, maybe you can even keep your placement in shape of a medal or badge on your profile...?
```
for consistency with the other bolds

> The aim of this league is to be an alternative, competitive and fun weekly activity for everyone to participate in while we use the maps pools to recommend people the best beatmaps osu! has to offer. This will give beatmaps in the Spotlight the attention they deserve even more.

Reviewer 2: ```suggestion
The aim of this league is to be an alternative, competitive and fun weekly activity for everyone to participate in while we use the mappools to recommend people the best beatmaps osu! has to offer. This will give beatmaps in the Spotlight the attention they deserve even more.
```

> The aim of this league is to be an alternative, competitive and fun weekly activity for everyone to participate in while we use the mapspools to recommend people the best beatmaps osu! has to offer. This will give beatmaps in the Spotlight the attention they deserve even more.

Reviewer: > recommend people the best beatmaps osu! has to offer

kinda repetitive with ur earlier paragraph. also typo "mapspools"

> For this, we also want to revamp the Spotlight Team as a whole. For that, we are looking for people willing and capable of selecting the best beatmaps created for all kinds of players so we can recommend them to everybody.

Reviewer 2: ```suggestion
To accomplish this, we also want to revamp the Spotlights Team as a whole. For that, we are looking for people willing and capable of selecting the best beatmaps created for players of all skill levels so we can recommend them to everybody.
```
Better flow of `For this, ____. For that, ____.`
Pretty sure by `kinds` you meant skill levels here, ignore this part if I am mistaken.

> **If you want to be part of the new Spotlight Team and help shaping this from the very beginning, [you can hand in your application now!](https://spotlights.team/app)**

Reviewer 2: ```suggestion
**If you want to be part of the new Spotlights Team and help shape this from the very beginning, [you can hand in your application now!](https://spotlights.team/app)**
```

> **If you want to be part of the new Spotlights Team and help shape this from the very beginning, [you can hand in your application now!](https://spotlights.team/app)**

Reviewer: same line break thing

We are also still looking for a good name for the league we want to add. You can leave your ideas in the comments below.

> With that, we hope that you like the new idea and that you can shape the new Spotlights together with us. **[Join us at our Discord server](https://discord.gg/DtQrzyV)** or visit the `#osu-spotlights` channel of osu!dev to help us with this project onwards!

Author: we are offering both options for now

> —Loctav & Kurokami

Reviewer: ```suggestion
—Loctav and Kurokami
```
seems like this is preferred because of how it's shown on the site
